src/index.tsx

```diff
@@ -3,15 +3,19 @@ import { createRoot } from 'react-dom/client';
 import { MockSolution, mockApi } from '@graasp/apps-query-client';
 
 import Root from './components/Root';
-import { ENABLE_MOCK_API, MODELS_ENDPOINT } from './config/settings';
+import {
+  ENABLE_MOCK_API,
+  MODELS_ENDPOINT,
+  MODEL_MOCK_ENDPOINT,
+} from './config/settings';
 import { MOCK_CONTEXT, buildDatabase } from './data/db';
 import './index.css';
 
 // setup mocked api for cypress or standalone app
 if (ENABLE_MOCK_API) {
   mockApi(
     {
-      externalUrls: [MODELS_ENDPOINT],
+      externalUrls: [MODELS_ENDPOINT, MODEL_MOCK_ENDPOINT],
```

Comment:
It should be possible to define external urls with wildcards `*`. So you do not need add a specific URL, instead you can say that anything that goes to a path with a wildcard should be treated as external. Try with:
```suggestion
externalUrls: [MODELS_ENDPOINT, `${MODEL_ENDPOINT}*`],
```